components/brave_rewards/resources/tip/components/tipRedditUser.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { bindActionCreators, Dispatch } from 'redux'
+import { connect } from 'react-redux'
+
+// Components
+import Banner from './siteBanner'
+import TransientTipOverlay from './transientTipOverlay'
+
+// Utils
+import * as rewardsActions from '../actions/tip_actions'
+import { getLocale } from '../../../../common/locale'
+
+interface Props extends RewardsTip.ComponentProps {
+  url: string
+  publisher: RewardsTip.Publisher
+  redditMetaData: RewardsTip.RedditMetaData
+}
+
+class TipRedditUser extends React.Component<Props, {}> {
+  get actions () {
+    return this.props.actions
+  }
+
+  render () {
+    const { finished, error } = this.props.rewardsDonateData
+
+    const publisher = this.props.publisher
+    const redditMetaData = this.props.redditMetaData
+    const key =
+      redditMetaData &&
+        redditMetaData.postText &&
+        redditMetaData.postText.length > 0
+        ? 'redditTipTitle'
+        : 'redditTipTitleEmpty'
+    publisher.title = 'u/' + getLocale(key, { user: redditMetaData.userName })
```

Comment:
shouldn't this be like `publisher.title = getLocale(key, { user: 'u/' + redditMetaData.userName })`?

Reply:
![image](https://user-images.githubusercontent.com/9574457/59910135-95cb6300-9411-11e9-8e6b-86f8fab7b42a.png)

Reply:
Fixed